index.html

```diff
@@ -159,92 +170,104 @@ <h3>About Me</h3>
+            </div>
+            <h3>Frameworks</h3>
+            <ul class="tech-learned">
+              <li class="tech-learned-item"><a href="#">Bootstrap</a></li>
+              <li class="tech-learned-item"><a href="#">Ruby on rails</a></li>
+              <li class="tech-learned-item"><a href="#">RSpec</a></li>
+              <li class="tech-learned-item"><a href="#">Capybara</a></li>
+              <li class="tech-learned-item"><a href="#">Selenium</a></li>
+            </ul>
           </div>
-          <h3>Frameworks</h3>
-          <ul class="tech-learned">
-            <li class="tech-learned-item"><a href="#">Bootstrap</a></li>
-            <li class="tech-learned-item"><a href="#">Ruby on rails</a></li>
-            <li class="tech-learned-item"><a href="#">RSpec</a></li>
-            <li class="tech-learned-item"><a href="#">Capybara</a></li>
-            <li class="tech-learned-item"><a href="#">Selenium</a></li>
-          </ul>
-        </div>
-        <div class="about-me-card">
-          <div class="img-container">
-            <img src="icons/icon-skills.svg" alt="Skills" />
+          <div class="about-me-card">
+            <div class="img-container">
+              <img src="icons/icon-skills.svg" alt="Skills" />
+            </div>
+            <h3>Skills</h3>
+            <ul class="tech-learned">
+              <li class="tech-learned-item"><a href="#">Codekit</a></li>
+              <li class="tech-learned-item"><a href="#">GitHub</a></li>
+              <li class="tech-learned-item"><a href="#">Codepen</a></li>
+              <li class="tech-learned-item"><a href="#">Gitlab</a></li>
+              <li class="tech-learned-item"><a href="#">Terminal</a></li>
+            </ul>
           </div>
-          <h3>Skills</h3>
-          <ul class="tech-learned">
-            <li class="tech-learned-item"><a href="#">Codekit</a></li>
-            <li class="tech-learned-item"><a href="#">GitHub</a></li>
-            <li class="tech-learned-item"><a href="#">Codepen</a></li>
-            <li class="tech-learned-item"><a href="#">Gitlab</a></li>
-            <li class="tech-learned-item"><a href="#">Terminal</a></li>
-          </ul>
         </div>
       </section>
 
+      <!-- FORM SECTION -->
+
       <form action="https://formspree.io/f/xoqbveqq" method="post">
         <h2>
           I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
         </h2>
         <ul class="form-ul">
-          <li class="form-li">
-            <input autocomplete="off" maxlength="30" type="text" name="name" id="name" placeholder="Full Name" required>
+          <li class="form-li fst-and-lst-nm-input">
+            <input autocomplete="off" maxlength="30" type="text" name="first-name" id="first-name" placeholder="First Name" required>
+            <input autocomplete="off" maxlength="30" type="text" name="last-name" id="last-name" placeholder="Last Name" required>
+          </li>
```

Comment:
- [ ] In the project requirements there was a note mentioning that on templates with different input fields from mobile to desktop, the 'full name' input field should be used for both versions. Please remove those fields and use only the 'full name' field for the desktop too. 👍

Reply:
May I leave the form like that? It's working, please let me know :))